Vish/assignment-6/style.css

```diff
@@ -0,0 +1,20 @@
+h2,h3{
+    margin: 0;
+    text-align: right;
+    font-family: Arial, Helvetica, sans-serif;
+}
+.box{
+    background-color: #8a9ba6;
+    border-radius:0 10px 0 10px;
+    margin: 20px 0 10px 0;
+
+}
+hr{
+    margin-top: 50px;
+}
+div span{
```

Comment:
use span only for font-weight not div, because in div tag you not write any text but you write text in heading tag.